tests/unit/Helmich/TypoScriptLint/Linter/ReportPrinter/GccReportPrinterTest.php

```diff
@@ -20,16 +20,15 @@
 class GccReportPrinterTest extends TestCase
 {
 
-    const EXPECTED_OUTPUT = 'foobar.tys:123:12: info: Message #1
+    public const EXPECTED_OUTPUT = 'foobar.tys:123:12: info: Message #1
 foobar.tys:124:0: warning: Message #2
 bar.txt:412:141: error: Message #3
 ';
 
-    /** @var BufferedOutput */
-    private $output;
+    private BufferedOutput $output;
 
     /** @var ConsoleReportPrinter */
```

Comment:
Don't these two conflict with each other? Since  `$this->printer` is always initialized with a `new GccReportPrinter` in the `setUp` method, this should (?) indubitably always be a `GccReportPrinter`.

```suggestion
```

Reply:
Done.

Reply:
(and rebased)